libraries/botbuilder/src/routeConstants.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * @module botbuilder
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+/* eslint-disable @typescript-eslint/no-namespace */
```

Comment:
I followed the standard set in [AuthenticationConstants](https://github.com/microsoft/botbuilder-js/blob/master/libraries/botframework-connector/src/auth/authenticationConstants.ts), but [eslint didn't like that](https://github.com/typescript-eslint/typescript-eslint/blob/v1.13.0/packages/eslint-plugin/docs/rules/no-namespace.md). Let me know if I should change it up.

Reply:
Let's go the CallerIdConstants route in `botframework-schema` instead.